packages/web/src/components/layouts/PackagePage/PackagePage.tsx

```diff
@@ -0,0 +1,596 @@
+                ) : (
+                  <SitesList sites={usedOnList} className={styles.usedOnList} />
+                )}
+              </section>
+
+              <section className={styles.versions}>
+                <div className={styles.versionsTop}>
+                  <h2 className={styles.versionsTitle}>Versions</h2>
+
+                  <button
+                    className={styles.mobileSort}
+                    type='button'
+                    onClick={() => setModalSortOpen(true)}
+                    disabled={loading}
+                  >
+                    <Icon
+                      kind='sort'
+                      width={14}
+                      height={14}
+                      color={loading ? '#8E8AA0' : '#212121'}
+                    />
+                  </button>
+
+                  <div className={styles.versionsActions}>
+                    <span className={styles.versionsActionsTitle}>Sort by</span>
+                    <span className={styles.sort}>
+                      {/* TODO: move sort to separate entity/component */}
+                      {sorts.map((name) => (
+                        <Button
+                          key={name}
+                          variant='secondary'
+                          size='medium'
+                          className={clsx(
+                            styles.sortButton,
+                            sortField === name && styles.sortButtonActive
+                          )}
+                          onClick={() => requestSort(name)}
+                          disabled={loading}
+                        >
+                          {name[0].toUpperCase() + name.slice(1)}
+
+                          {!loading &&
+                            (sortField === name ? (
+                              <Icon
+                                kind='arrowDown'
+                                width={10}
+                                height={12}
+                                className={clsx(
+                                  styles.sortIcon,
+                                  sortDirection === 'desc' && styles.sortIconDesc
+                                )}
+                              />
+                            ) : (
+                              <Icon
+                                kind='chevronDown'
+                                width={10}
+                                height={7}
+                                className={styles.sortIcon}
+                              />
+                            ))}
+                        </Button>
+                      ))}
+                    </span>
+                  </div>
+                </div>
+
+                <div className={styles.packages}>
+                  {loading ? (
+                    repeat(5, <Skeleton width='100%' height={100} variant='rounded' />)
+                  ) : (
+                    <>
+                      <PackageVersion
```

Comment:
Could you please move the data from here into `loadedData` and update render logic?